pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -58,8 +58,11 @@ type MonitoringStackSpec struct {
 	LogLevel LogLevel `json:"logLevel,omitempty"`
 
 	// Label selector for Monitoring Stack Resources.
+	// Set to the empty LabelSelector to monitoring everything.
```

Comment:
```suggestion
	// Set to an empty LabelSelector ({}) to monitoring everything.
```

Reply:
But isn't there just the one empty LabelSelector? Put :+1: on the `({})` bit.